src/main/resources/dbreplica/migration/V1.0.8__task_assignments_plsql.sql

```diff
@@ -0,0 +1,15 @@
+drop table if exists task_assignments cascade;
+create table cft_task_db.task_assignments
+(
+  assignment_id               SERIAL NOT NULL,
```

Comment:
Any reason this is a serial instead of just a uuid?  We probably have dates for query sorting.

Reply:
As discussed with Ravi, this will be refactored RWA-2085